### DISCUSSION TITLE
Add excel sheet with components Pick and Place

### DISCUSSION THREAD

**Discussion by Bebowl** [will127534/OneInchEye#18] [Ideas]
Hi Will,
First thanks for this great work.
I would like to know if it could be possible to add an excel spreadsheet with components pick and place in your project.
It will make PCB build much easier.
Thanks for your help

Ben

**Comment by will127534** [comment]
Have you checked the repo? It is under /Bom and /Gerber....

**Comment by Bebowl** [comment]
Ho.. My mistake, sorry for this lazy question and thanks for your answer. Must wake up...
